keyboards/novelkeys/nk87/post_rules.mk

```
@@ -0,0 +1,14 @@
+ifeq ($(strip $(RGB_BACKLIGHT)),yes)
+
+# Disable RGB Matrix
+RGB_MATRIX_ENABLE = no
```

Honestly, I'd opt for this: 
```suggestion
RGB_MATRIX_SUPPORTED = no
```

This actually disables the feature, even when enabling the feature from command line. It prevents it from being used *at all*.  Which is more appropriate for what this is trying to do.

---

@drashna, I don't follow it. Is it about to introduce introduce different detective (RGB_MATRIX_SUPPORTED) that will control what RGB should be used (RGB_MATRIX or RGB_BACKLIGHT)?

I am thinking to implement something like:

```
ifeq ($(filter $(strip $(RGBLED_OPTION_TYPE))x, nonex backlightx matrixx x),)
   $(error unknown RGBLED_OPTION_TYPE value "$(RGBLED_OPTION_TYPE)")
endif

ifeq ($(strip $(RGBLED_OPTION_TYPE)),backlight)
RGB_MATRIX_ENABLE = no
RGB_BACKLIGHT = yes

.......

endif
ifeq ($(strip $(RGBLED_OPTION_TYPE)),matrix)
RGB_BACKLIGHT = no
RGB_MATRIX_ENABLE = yes
RGB_MATRIX_DRIVER = IS31FL3733

..........

endif
```

So, both RGB options can be supported and configured at keymap level by setting the RGBLED_OPTION_TYPE.

---

> RGB_MATRIX_SUPPORTED = no

This actually won't work, because `disable_features.mk` is included before any of the `post_rules.mk` files.